### ISSUE TITLE
[BUG] Folder creation fails with trailing slash

### ISSUE THREAD

**Issue by robin-tribe29** [tribe29/ansible-collection-tribe29.checkmk#42]
**Describe the bug**

Trying to create a folder with a trailing slash lets the `folder` module fail.

**Component Name**

`folder`

**Ansible Version**

```console
$ ansible --version
ansible [core 2.12.5]
  config file = /vagrant/ansible.cfg
  configured module search path = ['/vagrant/plugins/modules']
  ansible python module location = /usr/local/lib/python3.8/dist-packages/ansible
  ansible collection location = /home/vagrant/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible
  python version = 3.8.10 (default, Mar 15 2022, 12:22:08) [GCC 9.4.0]
  jinja version = 2.10.1
  libyaml = True
```

**Checkmk Version**

```console
2.0.0p23 (CRE)
```

**Collection Version**

```console
$ ansible-galaxy collection list
# /home/vagrant/.ansible/collections/ansible_collections
Collection      Version
--------------- -------
tribe29.checkmk 0.2.2 
```
**Environment**

Ubuntu 20.04
Python 3.8.10

**To Reproduce**
1. Create a folder without a trailing slash
2. Try to create the exact same folder again, but with a trailing slash now

**Expected behavior**

We get an OK state from Ansible, as the folder already exists.

**Actual behavior**

We get an error message and the task fails:
```
TASK [Create folders.] **********************************************************************************
failed: [test1.tld -> localhost] (item={'path': '/test/', 'title': 'Test'}) => {"ansible_loop_var": "item", "changed": false, "item": {"path": "/test/", "title": "Test"}, "msg": "Error calling API. HTTP code 400. Details: b'{\"title\": \"Path already exists\", \"status\": 400, \"detail\": \"The path \\'\\\\/test\\' already exists.\"}', "}
```

**Screenshots**

**Additional context**

**Comment by robin-tribe29** [comment]
There are changes that need to be made to the Checkmk REST API, which we are waiting for.

**Comment by lgetwan** [comment]
This is fixed in the upcoming 2.1.0p5 (already in current 2.1.0 branch's daily build)

**Comment by robin-tribe29** [comment]
Closing, as this has long been fixed.
